drivers/node/aws/aws.go

```diff
@@ -0,0 +1,128 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/portworx/torpedo/drivers/node"
+	//"github.com/portworx/torpedo/drivers/scheduler"
+	aws_pkg "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"log"
+)
+
+const (
+	// DriverName is the name of the aws driver
+	DriverName = "aws"
+	// AwsRegion is the region of aws ec2
+	AwsRegion = "us-east-1"
+)
+
+type aws struct {
+	node.Driver
+	session     *session.Session
+	credentials *credentials.Credentials
+	config      *aws_pkg.Config
+	svc         *ec2.EC2
+	Instances   []*ec2.Instance
+}
+
+func (aws *aws) String() string {
+	return DriverName
+}
+
+func (aws *aws) Init(sched string) error {
+	sess := session.Must(session.NewSession())
+	aws.session = sess
+	creds := credentials.NewEnvCredentials()
+	aws.credentials = creds
+	config := &aws_pkg.Config{Region: aws_pkg.String(AwsRegion)}
+	config.WithCredentials(creds)
+	aws.config = config
+	svc := ec2.New(sess, config)
+	aws.svc = svc
+	//Get all nodes from east region
+	instances := aws.getAllInstancesInEast()
+	aws.Instances = instances
+	return nil
+}
+
+func (aws *aws) TestConnection(n node.Node, options node.TestConectionOpts) error {
+	return nil
+}
+
+func (aws *aws) RebootNode(n node.Node, options node.RebootNodeOpts) error {
+	var err error
+	_, err = aws.getAddrToConnect(n)
```

Comment:
`getAddrToConnect` isn't required for the aws driver. There isn't any useful logic being performed by this routine. Remove it.

Reply:
removed